### PR TITLE
adobe-air-sdk: migrate to Caskroom

### PR DESCRIPTION
**Pull request by adidalal**
https://github.com/caskroom/homebrew-cask/issues/23938

Cask at https://github.com/caskroom/homebrew-cask/pull/23942

@MikeMcQuaid is this the right way to do things?

### CODE REVIEW

File: tap_migrations.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+"adobe-air-sdk": "caskroom/cask"
```
Comment (MikeMcQuaid) [inline tap_migrations.json]: Pedantry: indent by two spaces.

**Comment by MikeMcQuaid** [comment]
@adidalal One comment otherwise 👍

**Comment by adidalal** [comment]
Updated

**Comment by MikeMcQuaid** [comment]
Thanks @adidalal!

**Pull request merged**